docs/recipes/travis_build_stages.md

```diff
@@ -0,0 +1,90 @@
+# Using semantic-release with [Travis build stages](https://docs.travis-ci.com/user/build-stages/)
+
+![Travis Build Stages screenshot](https://cloud.githubusercontent.com/assets/3729517/25229553/0868909c-25d1-11e7-9263-b076fdef9288.gif)
+
+Build stages are a great way to run semantic release compared to running it in `after_success`:
+
+* The release stage can use Node 8, while the other stages can use other versions, even use a completely different language.
+* It allows you to easily release a project that is not even written in JavaScript, without impacting the job configuration of your test jobs in any way.
```

Comment:
Can Travis runs `npx semantic-release` on a non Node image? If yes this comment is irrelevant.

Reply:
Not afaik because npx was introduced in npm 5. Also are you sure that every image has Node preinstalled?

Reply:
Ok, let's forget about `npx`